flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -0,0 +1,90 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for the Flocker Docker plugin.
+"""
+
+import ssl
+from twisted.trial.unittest import TestCase
+
+from docker import Client
+from docker.tls import TLSConfig
+from docker.utils import create_host_config
+
+from ...testtools import random_name
+from ..testtools import (
+    require_cluster, post_http_server, assert_http_server,
+)
+from ..obsolete.test_containers import CURRENT_DIRECTORY
```

Comment:
This is what happens when I have to continuiously context switch.

`obsolete` is for stuff that we are supporting only because we have to, in particular container API. I will move stuff out of there, yeah.